Makefile

```diff
@@ -157,9 +157,12 @@ verify: .init .generate_files
 	@rm .out
 	@#
 	@echo Running golint and go vet:
-	# Exclude the generated (zz) files for now
+	@# Exclude the generated (zz) files for now, as well as defaults.go (it
```

Comment:
Not a show stopper, but I would prefer to not hide this comment - I'd like it to be a nagging reminder for someone to fix it upstream.